packages/fx-core/src/plugins/resource/bot/question.ts

```diff
@@ -42,22 +42,22 @@ export const AppServiceOptionItem: HostTypeTriggerOptionItem = optionWithL10n({
 });
 
 // TODO: this option will not be shown in UI, leave messages empty.
-export const AppServiceOptionItemForVS: HostTypeTriggerOptionItem = {
+export const AppServiceOptionItemForVS: HostTypeTriggerOptionItem = optionWithL10n({
   id: "http-webapi",
   hostType: HostTypes.APP_SERVICE,
-  label: "",
-  cliName: "",
-  description: "",
-  detail: "",
-};
+});
 
 export const HostTypeTriggerOptions: HostTypeTriggerOptionItem[] = [
   AppServiceOptionItem,
   FunctionsHttpTriggerOptionItem,
   FunctionsTimerTriggerOptionItem,
 ];
 
-export const HostTypeTriggerOptionsForVS: HostTypeTriggerOptionItem[] = [AppServiceOptionItemForVS];
+export const HostTypeTriggerOptionsForVS: HostTypeTriggerOptionItem[] = [
```

Comment:
Since FunctionsHttpTriggerOptionItem and FunctionsTimerTriggerOptionItem are common between VS and VSCode, it looks like there could be HostTypeTriggerOptionItem[] including these 2 items. And then we add DefaultOptionItem by platform to that HostTypeTriggerOptionItem[], instead of creating 2 const array?

Reply:
> ![image](https://user-images.githubusercontent.com/25220706/173509819-64f1d77f-a4c2-4834-ab60-15bc239d98df.png)

For the Trigger strings/names, we can keep align with VS Code, by doing that, just need to change "webapi Server" to "Restify Server". The remainings looks good.

Reply:
thanks for clarify, then use Webapi will be good.

Reply:
> Since FunctionsHttpTriggerOptionItem and FunctionsTimerTriggerOptionItem are common between VS and VSCode, it looks like there could be HostTypeTriggerOptionItem[] including these 2 items. And then we add DefaultOptionItem by platform to that HostTypeTriggerOptionItem[], instead of creating 2 const array?

Done.